lib/browser/rpc-server.js

```diff
@@ -114,7 +114,7 @@ ipcMainUtils.handleSync('ELECTRON_BROWSER_SANDBOX_LOAD', async function (event)
     process: {
       arch: process.arch,
       platform: process.platform,
-      env: process.env,
+      env: { ...process.env },
```

Comment:
`process.env` is a C++-backed "special" JS object, so it can't be serialized by Structured Clone. This code would work unmodified currently, but it would fall back on the old `base::Value` serialization and emit a deprecation warning, and begin throwing an exception in Electron 9.

Reply:
Sounds good, thanks for the explanation. I didn't realize `process.env` was a special case compared to other objects.